spec/integration/chunker_spec.rb

```diff
@@ -71,9 +71,9 @@ def printer.end(*) [] ;end
     it 'should throttle work stride based on slave lag' do
       5.times { |n| execute("insert into origin set id = '#{ (n * n) + 1 }'") }
 
-      printer = MiniTest::Mock.new
-      15.times { printer.expect(:notify, :return_value, [Fixnum, Fixnum]) }
-      printer.expect(:end, :return_value, [])
+      printer = mock()
+      printer.expects(:notify).with(instance_of(Fixnum), instance_of(Fixnum)).twice
```

Comment:
Why is this `.twice` when the old code was `15.times`?

(I mean, `.twice` looks correct though, with `max_retries` being two).